class.jetpack.php

```diff
@@ -3233,14 +3233,14 @@ public static function unlink_user( $user_id = null ) {
 		if ( ! isset( $tokens[ $user_id ] ) )
 			return false;
 
-		Jetpack::load_xml_rpc_client();
-		$xml = new Jetpack_IXR_Client( compact( 'user_id' ) );
-		$xml->query( 'jetpack.unlink_user', $user_id );
+		if ( $bypass_wpcom ) {
```

Comment:
do you mean `! $bypass_wpcom` here? Right now by default, this method will unlink a user and not tell wpcom.  We're using this method currently for unlinking.